.github/workflows/r_nightly.yml

```diff
@@ -101,18 +127,37 @@ jobs:
           # strip superfluous nested dirs
           new_paths <- sub(art_path, ".", new_paths)
           dirs <- dirname(new_paths)
-          dir_result <- sapply(dirs, dir.create, recursive = TRUE)
-
-          if (!all(dir_result)) {
-            stop("There was an issue while creating the folders!")
-          }
+          sapply(dirs, dir.create, recursive = TRUE, showWarnings = FALSE)
 
-          copy_result <- file.copy(current_path, new_paths)
+          # overwrite allows us to "force push" a new version with the same name
+          copy_result <- file.copy(current_path, new_paths, overwrite = TRUE)
 
           if (!all(copy_result)) {
             stop("There was an issue while copying the files!")
           }
+      - name: Prune Repository
+        shell: bash
+        env:
+          KEEP: ${{ github.event.inputs.keep || 14 }}

```

Comment:
shouldn't this already be the default value? I am not sure I understand why do we need the `|| 14`